src/controllers/auth/login.ts

```diff
@@ -0,0 +1,28 @@
+import * as Interfaces from "@interfaces";
+import * as Utils from "@utils";
+
+const login: Interfaces.Middlewares.Async = async (req, res, next) => {
+  const body: Interfaces.User = req.body;
+  try {
```

Comment:
No need to use try-catch. Error handler catches all the errors that happen mid-way into a controller (the control flow stops and looks for the control handler)

Reply:
oh okay 👍🏻